content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md

```diff
@@ -238,7 +238,7 @@ Follow [these steps](/docs/tasks/traffic-management/egress/egress-gateway-tls-or
 1. Create required `RBAC` to make sure the secret created in the above step is accessible to the client pod, which is `sleep` in this case.
 
     {{< text bash >}}
-    $ kubectl create role client-credential-role --resource=secret --verb=list
+    $ kubectl create role client-credential-role --resource=secret --verb=get --resource-name=client-credential
```

Comment:
Please check: https://github.com/istio/istio/blob/master/pilot/pkg/credentials/kube/secrets.go#L156 This won't work. I can run the documentation test for you on this PR and show that this will fail.

Reply:
Thanks for pointing this out. However with this narrow role, my setup still works interestingly.
Could you run the tests and let's see if it will fail or not?

Reply:
All right, I was able to reproduce the error myself. Closing the PR then.